### ISSUE TITLE
Compilation fails on Scientific Linux 7.1

### ISSUE THREAD

**Issue by timando** [jgeboski/purple-facebook#10]
I've installed the required prerequisites, but the autogen script fails with the following output.

```
$ ./autogen.sh
pulling from https://hg.pidgin.im/soc/2015/jgeboski/facebook
searching for changes
no changes found
resolving manifests
0 files updated, 0 files merged, 0 files removed, 0 files unresolved
patching file libpurple/protocols/facebook/Makefile.am
patching file libpurple/protocols/facebook/Makefile.mingw
patching file libpurple/protocols/facebook/facebook.c
patching file libpurple/protocols/facebook/facebook.h
patching file libpurple/http.c
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -Im4
configure.ac:88: error: AC_REQUIRE(PKG_PROG_PKG_CONFIG): cannot be used outside of an AC_DEFUN'd macro
configure.ac:88: the top level
autom4te: /usr/bin/m4 failed with exit status: 1
aclocal: error: echo failed with exit status: 1
autoreconf: aclocal failed with exit status: 1
```

**Comment by jgeboski** [comment]
Do you have pkg-config installed?  If so, what version?

**Comment by timando** [comment]
yes, I have version 0.27.1 installed.

**Comment by jgeboski** [comment]
I have a patch which should resolve this.  Please test it for me just to ensure it does in fact fix the issue.

```
wget -O- https://gist.githubusercontent.com/jgeboski/a815c00337a998a5f36d/raw/167b027dd7f7a69d09988597eb77eb615929027f/04-pkgconfig.patch | patch -p1
./autogen.sh
```

**Comment by timando** [comment]
That fixed it.

**Comment by jgeboski** [comment]
Fixed as of 6895215.
